### PR TITLE
Hum 50

### PR DESCRIPTION
**Pull request by KacperKoza343**
task HUM-50

### CODE REVIEW

File: packages/apps/human-app/frontend/src/api/fetcher.ts [hunk 1]
```diff
@@ -99,10 +102,18 @@ export function createFetcher(defaultFetcherConfig?: {
       : fetcherOptions.options;
     if (fetcherOptions.authenticated) {
       fetcherOptionsWithDefaults = appendHeader(fetcherOptionsWithDefaults, {
-        Authorization: `Bearer ${browserAuthProvider.getAccessToken() ?? ''}`,
+        // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
```
Comment (dnechay) [inline packages/apps/human-app/frontend/src/api/fetcher.ts:105]: Wouldn't it be better if we change the rule globally in order to allow not only strings, but also booleans, numbers and null/undefined?

**Review by dnechay — COMMENTED** [review]
Approach lgtm, 2 points

1) We should apply the same logic to oracles table queries (now you can spawn multiple requests by selecting in job type filter)
2) Make sure your git history is clean before we merge
3) Do we need this PR? It seems to be part of https://github.com/humanprotocol/human-protocol/pull/2661

**Comment by KacperKoza343** [comment]
> Approach lgtm, 2 points
> 
> 1. We should apply the same logic to oracles table queries (now you can spawn multiple requests by selecting in job type filter)
> 2. Make sure your git history is clean before we merge
> 3. Do we need this PR? It seems to be part of [Hum 40 hum 50 #2661](https://github.com/humanprotocol/human-protocol/pull/2661)

1. I will add it there
2. Done
3. Yes, that's right, but HUM-40 also have debounce so I added this one separately to have a better ability to test it, with debounce it's hard to see cancelled requests

**Comment by dnechay** [comment]
@KacperKoza343 
> so I added this one separately to have a better ability to test it

That's true, but you always can comment some code in your working branch or alter git history for testing purpose.

If we keep two separate PRs and merge this one first - it will create unnecessary difficulties for you in the second one with resolving conflicts.

Up to you in the end

File: packages/apps/human-app/frontend/src/api/fetcher.ts (hunk 1 above)
Comment (dnechay) [reply packages/apps/human-app/frontend/src/api/fetcher.ts:105]: @KacperKoza343 could you please do it in this PR?

**Review by dnechay — COMMENTED** [review]
Looks good, one comment about linter comment

File: packages/apps/human-app/frontend/src/api/fetcher.ts (hunk 1 above)
Comment (KacperKoza343) [reply packages/apps/human-app/frontend/src/api/fetcher.ts]: done

**Review by dnechay — APPROVED** [review]

**Pull request merged**